sphinx/ext/autosummary/templates/autosummary/module.rst

```diff
@@ -34,3 +34,15 @@
    {%- endfor %}
    {% endif %}
    {% endblock %}
+
+{% block modules %}
+{% if modules %}
+.. rubric:: modules
+
+.. autosummary::
+   :toctree: modules
```

Comment:
Is output directory fixed?

Reply:
What else do you suggest? Some options one can think of:
 - specify nothing (`:toctree:`): all files are generated in the same directory
 - use package name (`:toctree: {{ name }}`): might be confusing (dir tree does not reflect package/subpackage structure)
 - use subpackage name (`:toctree: ???`): might be confusing (dir tree does not reflect package/subpackage structure). Add another context variable `generate_autosummary_content`.
 - inherit parent toctree (`:toctree: ???`): Add another context variable `generate_autosummary_content`. Not sure how to retrieve the parent toctree name.

Reply:
At I commented first, I thought inheriting parent toctree is better. But, now, I feel it seems no reasons to make a sub directory. Why `generated/modules/modules/modules/...` directories are needed? IMO, same directory is better to me.